### PR TITLE
[Bootstrap Admin Panel] Order's Address Edit Page View

### PR DESCRIPTION
**Pull request by Rafikooo**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | bootstrap-admin-address                   |
| Bug fix?        | no                                                       |
| New feature?    | kinda                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no   |
| License         | MIT                                                          |

image

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/views/Order/_form.html.twig
Comment (jakubtobiasz) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Order/_form.html.twig:1]: I'm not changing it yet (however I did it while creating new templates under the `Common` directory. Wdyt about dropping the `_` prefix? It comes from the partials, but here I don't see any value in it.

File: src/Sylius/Behat/Page/Admin/Order/UpdatePage.php [hunk 1]
```diff
@@ -63,6 +63,43 @@ public function checkValidationMessageFor(string $element, string $message): boo
...
+    private function getOptionTextsFor(NodeElement $element): array
+    {
+        $result = [];
+        $options = $element->findAll('css', 'option');
+
+        foreach ($options as $option) {
+            $result[] = $option->getText();
+        }
+
+        return $result;
```
Comment (NoResponseMate) [inline src/Sylius/Behat/Page/Admin/Order/UpdatePage.php]:
```suggestion
        return array_map(fn ($option) => $option->getText(), $element->findAll('css', 'option'));
```

File: src/Sylius/Bundle/AdminBundle/Resources/views/Order/_form.html.twig
Comment (NoResponseMate) [reply src/Sylius/Bundle/AdminBundle/Resources/views/Order/_form.html.twig:1]: It is a partial, but its reusability is pretty much none, so I'm for it

File: src/Sylius/Bundle/AddressingBundle/Form/Type/AddressType.php [hunk 2]
```diff
@@ -78,6 +81,7 @@ public function configureOptions(OptionsResolver $resolver): void
                     return $this->validationGroups;
                 },
                 'shippable' => false,
+                'add_build_address_form_subscriber' => true,
```
Comment (Wojdylak) [inline src/Sylius/Bundle/AddressingBundle/Form/Type/AddressType.php:84]: :thinking: Add information about this option to the readme, WDYT?

File: src/Sylius/Bundle/AdminBundle/Resources/views/Order/_form.html.twig
Comment (jakubtobiasz) [reply src/Sylius/Bundle/AdminBundle/Resources/views/Order/_form.html.twig:1]: Cool, I've removed all underscores in this PR, and I'll apply this rule to the already existing templates later. 

Maybe I'll write some ADR (or a similar document) too.

**Review by TheMilek — APPROVED** [review]

File: src/Sylius/Bundle/AddressingBundle/Form/Type/AddressType.php (hunk 2 above)
Comment (jakubtobiasz) [reply src/Sylius/Bundle/AddressingBundle/Form/Type/AddressType.php:84]: 1) It's minor change
2) It doesn't change a lot, as we added it for the new feature, previously behavior has been kept
3) We might pivot into separate form types in the meantime

**Review by Wojdylak — APPROVED** [review]

**Pull request merged**